packages/lab/src/components/Flow/DroppableFlow.tsx

```diff
@@ -235,6 +236,15 @@ export const HvDroppableFlow = ({
 
   const validEdges = validateEdges(edges, nodes, nodeTypes);
 
+  const defaultEdgeOptions = {
```

Comment:
I don't know if I'm reading too much into this 😅 but maybe we should do something like shown below in order for the `markerEnd` values we have defined to not be overwritten if someone defines the `markerEnd` on `defaultEdgeOptions` just to change the `color` let's say:

```
 const defaultEdgeOptions = {
     markerEnd: {
       type: MarkerType.ArrowClosed,
       height: 20,
       width: 20,
       ...defaultEdgeOptionsProp?.markerEnd,
     },
     ...defaultEdgeOptionsProp,
 };
```

Reply:
As discussed, we'll leave this as it is since the `markerEnd` can be either a string or an object.